### PR TITLE
Docs for weld-porosity-detection-0001

### PR DESCRIPTION
**Pull request by druzhkov-paul**

### CODE REVIEW

**Comment by druzhkov-paul** [comment]
@snosov1 @vladimir-dudnik

File: demos/python_demos/action_recognition/models.lst [hunk 1]
```diff
@@ -3,3 +3,4 @@ action-recognition-????-decoder
 action-recognition-????-encoder
 driver-action-recognition-adas-????-decoder
 driver-action-recognition-adas-????-encoder
+weld-porosity-detection-0001
```
Comment (Wovchena) [inline demos/python_demos/action_recognition/models.lst:6]:
```suggestion
weld-porosity-detection-????
```

File: demos/python_demos/action_recognition/weld_defects.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+no weld
+normal weld
+porosity
+
```
Comment (Wovchena) [inline demos/python_demos/action_recognition/weld_defects.txt:4]:
```suggestion
```

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/python_demos/action_recognition/models.lst [hunk 3]
```diff
@@ -3,3 +3,4 @@ action-recognition-????-decoder
 action-recognition-????-encoder
 driver-action-recognition-adas-????-decoder
 driver-action-recognition-adas-????-encoder
+weld-porosity-detection-????
```
Comment (IRDonch) [inline demos/python_demos/action_recognition/models.lst:6]: You'll need to remove this for now, as this'll make the demo tests fail until we publish the actual model files. I'll try to readd it when I do that.

**Pull request merged**